docs/changes/coordinates/11690.other.rst

```diff
@@ -0,0 +1,3 @@
+Sphinx RST links in ``SkyCoord`` now use intersphinx for downstream packages' convenience.
```

Comment:
Isn't it all through astropy now? Or is `SkyCoord` still the only thing that could be subclassed and thus be wrong?

Reply:
yes, it's now all over astropy. I'm moving the note up to the top level. I've contributed a number of such fragments, all about xref, docstrings, and the documentation. Perhaps there should be a specific category / folder for such changelog fragmets, e.g. ``documentation/``.   @saimn how feasible is this with towncrier?

Reply:
It's easy to do but until now the policy has been that we do not mention documentation changes. Exceptions are possible such as for big changes like you did but I'm not sure that justifies a new category. "other" seems fine to me. @pllim @bsipocz - Thoughts ?

Reply:
I don't think this would need a changelog to begin with. And as a rule of thumb, the changelog doesn't have any links. The links tend to broke on the many year timescale, and while SkyCoord is expected not to be one of those, the blanket rule of no links works better than trying to maintain a list of OK and not OK types of links.